### PR TITLE
Remove several deprecated utilities in the web package

### PR DESCRIPTION
**Pull request by dkanada**
The next step would be getting the web client to redirect to the startup wizard on its own rather than rely on the server for that step, and verify that the wizard can only be run once.

### CODE REVIEW

**Comment by dkanada** [comment]
This will require `apploader.js` and `appMode='standalone'` injected in the index. Basically, the standalone client will become the only flavor we need.

File: MediaBrowser.WebDashboard/Api/PackageCreator.cs [hunk 1]
```diff
@@ -1,161 +0,0 @@
...
-            using (var reader = new StreamReader(sourceStream, Encoding.UTF8))
-            {
-                html = await reader.ReadToEndAsync().ConfigureAwait(false);
-            }
-
-            if (isMainIndexPage && !string.IsNullOrWhiteSpace(localizationCulture))
-            {
-                var lang = localizationCulture.Split('-')[0];
-
-                html = html.Replace("<html", "<html data-culture=\"" + localizationCulture + "\" lang=\"" + lang + "\"", StringComparison.Ordinal);
```
Comment (JustAMan) [inline MediaBrowser.WebDashboard/Api/PackageCreator.cs:80]: who's going to do this now?

File: MediaBrowser.WebDashboard/Api/PackageCreator.cs [hunk 2]
```diff
@@ -1,161 +0,0 @@
...
-            }
-
-            if (isMainIndexPage)
-            {
-                html = html.Replace("<head>", "<head>" + GetMetaTags(mode), StringComparison.Ordinal);
-            }
-
-            // Disable embedded scripts from plugins. We'll run them later once resources have loaded
```
Comment (JustAMan) [inline MediaBrowser.WebDashboard/Api/PackageCreator.cs:88]: isn't this going to break something?

File: MediaBrowser.WebDashboard/Api/PackageCreator.cs [hunk 3]
```diff
@@ -1,161 +0,0 @@
...
-        /// <param name="version">The version.</param>
-        /// <returns>System.String.</returns>
-        private static string GetCommonJavascript(string mode, string version)
-        {
-            var builder = new StringBuilder();
-
-            builder.Append("<script>");
-            if (!string.IsNullOrWhiteSpace(mode))
-            {
-                builder.AppendFormat(CultureInfo.InvariantCulture, "window.appMode='{0}';", mode);
```
Comment (JustAMan) [inline MediaBrowser.WebDashboard/Api/PackageCreator.cs:135]: where is this handled/used?

Comment (dkanada) [reply MediaBrowser.WebDashboard/Api/PackageCreator.cs:135]: This is automatically set for the standalone flavor.

File: MediaBrowser.WebDashboard/Api/PackageCreator.cs (hunk 1 above)
Comment (dkanada) [reply MediaBrowser.WebDashboard/Api/PackageCreator.cs:80]: I should probably update the CI to pull the standalone build in this pull request. The language should be handled entirely based on what a user has for their browser settings, which I honestly prefer to this method.

File: MediaBrowser.WebDashboard/Api/PackageCreator.cs (hunk 2 above)
Comment (dkanada) [reply MediaBrowser.WebDashboard/Api/PackageCreator.cs:88]: I tested plugins after removing this and noticed no difference, but more extensive testing is always warranted.

**Comment by Artiume** [comment]
@Bond-009 this broke nightly due to missing an associated web PR.

**Comment by Bond-009** [comment]
Ok.... and?

**Comment by MrTimscampi** [comment]
We're handling the fix to this over on web. We simply weren't expecting it so soon :p

**Pull request merged**